static/app/components/smartSearchBar/searchBarDatePicker.tsx

```diff
@@ -176,6 +175,14 @@ const TimeInput = ({time, setTime}: TimeInputProps) => {
   );
 };
 
+const StyledOverlay = styled(Overlay)`
```

Comment:
IMO 
```suggestion
const SearchBarOverlay = styled(Overlay)`
```

is good

Reply:
Gotcha! Changed in https://github.com/getsentry/sentry/pull/38357/commits/a7a2dc038d56113341a160d093bddafc2f575a9d